invenio_records_marc21/services/services.py

```diff
@@ -85,6 +90,47 @@ def update_draft(
         data = self._create_data(identity, data, metadata, access)
         return super().update_draft(id_, identity, data, revision_id)
 
+    def _lift_embargo_from(self, record):
+        """Lifts embargo from record or draft."""
+        if not record.access.embargo.lift():
```

Comment:
where is the lift method defined?

Reply:
The lift function is here  [https://github.com/tu-graz-library/invenio-records-marc21/blob/master/invenio_records_marc21/records/systemfields/access/embargo.py#L45](here) defined.